docs/integrations/enterprise-connectors/source-oracle.md

```diff
@@ -0,0 +1,230 @@
+# Source Oracle
```

Comment:
Copying the contents of the markdown from the source of truth and pasting them here means we're going to be in a split-brain situation, but that seems like a reasonable trade-off.

Reply:
Yeah... one day we'll make a github action to keep these in sync

Reply:
I was thinking more that this content should be served straight from the metadata service or whatever it is that serves the docs pane when creating a connector in the app.

Reply:
Connector Ops had a dream of changing the docs side to use the markdown in the metadata service, but docusourus _really_ wants to prebuild all markdown files at build-time, from real markdown files.  I think it will be easier to keep things in sync git-wise rather than pull down content from the metadata service.  The build process is also triggerd by files change in git, which works well when things are updated in the other repo